test/unit/views/components/config/IleQueryFeatureConfigSpec.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.components.config
+import base.UnitSpec
+import config.AppConfig
+import org.mockito.Mockito.when
+import org.scalatest.BeforeAndAfterEach
+
+trait IleQueryFeatureConfigSpec extends UnitSpec with BeforeAndAfterEach {
```

Comment:
I think that extending `UnitSpec` here is an overkill. `MockitoSugar` should be sufficient.

Reply:
I find that's the problem with mixing in all these traits - working out which one you really need.  So I go with the one that works!   But I'll see if you're right.